base/_variables.scss

```diff
@@ -1 +1,15 @@
-// @file variables.scss
+// Variables throughout the project
```

Comment:
You think these belong globally? I think that we still need to look at further extrapolating out the responsive/typography stuff maybe into a base library as its a pretty significant piece of out scaffold.